test/e2e/perftype/perftype.go

```diff
@@ -18,6 +18,11 @@ package perftype
 
 // TODO(random-liu): Replace this with prometheus' data model.
 
+// CurrentKubeletPerfMetricsVersion is the current kubelet performance metrics
+// version. This is used by mutiple perf related data structures. We should
+// bump up the version each time we make an incompatible change to the metrics.
+const CurrentKubeletPerfMetricsVersion = "v2"
```

Comment:
This is also used in node e2e test. See line 34 at `test/e2e_node/benchmark_util.go`.